pyproject.toml

```diff
@@ -39,11 +39,11 @@ tests = [
     'phonopy==2.11.0',
     'h5py==3.6.0',
     'typing-extensions==4.4.0',
-    'ruff==0.1.4',
+    'ruff==0.1.8',
```

Comment:
Can v0.2.2 be used instead? I'm not sure of the editor integrations working in a submodule setup though.

Reply:
I can try, but I'd like to keep it the same to the main NOMAD\

Reply:
Yeah that's okay, you can leave it as is.